NAMESPACE

```diff
@@ -1,26 +1,4 @@
 # Generated by roxygen2: do not edit by hand
 
-export(swe_azalt)
-export(swe_calc)
-export(swe_close)
-export(swe_day_of_week)
-export(swe_deltat)
-export(swe_deltat_ex)
-export(swe_fixstar2)
-export(swe_fixstar2_mag)
-export(swe_get_tid_acc)
-export(swe_heliacal_ut)
-export(swe_lun_eclipse_how)
-export(swe_lun_eclipse_when)
-export(swe_lun_eclipse_when_loc)
-export(swe_pheno_ut)
-export(swe_rise_trans)
-export(swe_rise_trans_true_hor)
-export(swe_set_delta_t_userdef)
-export(swe_set_ephe_path)
-export(swe_set_tid_acc)
-export(swe_set_topo)
-export(swe_sol_eclipse_when_loc)
-export(swe_version)
```

Comment:
Oops, removing all the exports is not a good idea. Please inspect the changes before you commit them.